src/write-results-to-db.js

```diff
@@ -0,0 +1,110 @@
+const knex = require("knex")
+const moment = require("moment-timezone")
+const config = require("./config")
+
+const writeResultsToDatabase = (results, { realtime } = {}) => {
+  const db = knex({ client: "pg", connection: config.postgres })
+
+  if (realtime) {
+    return _writeRealtimeResults({ db, results }).then(() => db.destroy())
+  } else if (results.query.dimensions.match(/ga:date/)) {
+    return _writeRegularResults({ db, results }).then(() => db.destroy())
```

Comment:
Is `ga:date` a reliable distinguisher between the Real Time API and the Core Reporting API? I I believe the `query` object should have this information in a structured fashion, somewhere in its fields.